lib/matplotlib/backends/_backend_gtk.py

```diff
@@ -109,6 +112,143 @@ def _on_timer(self):
             return False
 
 
+class _FigureManagerGTK(FigureManagerBase):
+    """
+    Attributes
+    ----------
+    canvas : `FigureCanvas`
+        The FigureCanvas instance
+    num : int or str
+        The Figure number
+    toolbar : Gtk.Toolbar or Gtk.Box
+        The toolbar
+    vbox : Gtk.VBox
+        The Gtk.VBox containing the canvas and toolbar
+    window : Gtk.Window
+        The Gtk.Window
+    """
+
+    def __init__(self, canvas, num):
+        gtk_ver = Gtk.get_major_version()
```

Comment:
```suggestion
        self._gtk_ver = Gtk.get_major_version()
```
Do you want to store this on the manager so you can reuse it below instead of calling this method in other functions too?

Reply:
Sure.